war/src/main/js/widgets/add/remodal.less

```diff
@@ -0,0 +1,408 @@
+/*
+ *  Remodal - v1.0.7
```

Comment:
@recena I asked @kzantow and installWizard doesn't use a modal library, but just appears to.